src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/ActiveLicense.cs

```diff
@@ -20,15 +20,22 @@ public static ActiveLicenseFindResult Find(string applicationName, params Licens
             var licenseSourceResultToUse = LicenseSourceResult.DetermineBestLicenseSourceResult(results.ToArray());
             if (licenseSourceResultToUse != null)
             {
-                activeLicense.Report.Add($"Selected active license from {licenseSourceResultToUse.Location}");
+                var selectedLicenseReportItem = $"Selected active license from {licenseSourceResultToUse.Location}";
+                activeLicense.Report.Add(selectedLicenseReportItem);
```

Comment:
the non-breaking behavior makes sense as an argumentation, although not sure why not introduce breaking changes (and maybe also rename the report property) to make the change apparent to dependent code. It's not that we would follow semver in the licensing code, do we?